### ISSUE TITLE
Gridgen in Python

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/gridgen#1]
Convert gridgen matlab scripts to python.

**Comment by wxguy** [comment]
I agree with @JessicaMeixner-NOAA. This package should be ported to Python to reach wider audience. 

In my case, I wanted to try WW-III on my personal PC. But not able to do so due to non availability of Matlab software.

This link may be relevant https://github.com/metocean/pysmc.
